pages/src/App.js

```diff
@@ -11,39 +11,54 @@ class App extends Component {
   }
 
   componentWillMount () {
-    fetch('/todos').then(res =>
-      res.json().then(todos => {
-        this.setState({
-          todos: new Map().withMutations(map => {
+
+    // Create WebSocket connection.
+    let socket = new WebSocket('ws://localhost:3000');
+    this.socket = socket
```

Comment:
You should properly close this socket on [`componentWillUnmount`](https://devdocs.io/react/react-component#componentwillunmount).